views/chat.hbs

```diff
@@ -1,10 +1,61 @@
 <!DOCTYPE html>
-<html>
+<html lang="en">
   <head>
     <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/semantic-ui@2.2.13/dist/semantic.min.css">
     <title>Chat: Omeclone</title>
   </head>
-  <body>
-    Chat template where the strangers talk.
+  <style>
+    .seven{
```

Comment:
Why do you want to use custom styling? Semantic UI has great components. Please remove this and do it with semantic classes.